packages/template-master-detail-kinvey/README.md

```diff
@@ -35,36 +35,40 @@ tns create my-master-detail-kinvey --template nativescript-app-templates/package
 
 ### Kinvey integration
-The templates uses the [{N} Kinvey plugin](https://github.com/Kinvey/nativescript-sdk). The initialization is done before the app starts in the `/app.js` file. The initialization script is located at `/shared/kinvey.common.js`.
+The templates uses the [{N} Kinvey plugin](https://github.com/Kinvey/nativescript-sdk). The initialization is done before the app starts in the `/app/app.js` file. The initialization script is located at `/app/shared/kinvey.common.js`.
 
 ### [Optional] Kinvey database setup
 By design the app is connected to a read-only copy of the sample data in Kinvey. If you want to see the "edit" functionality in action you will have to clone the sample data and update the app configuration to point to your own Kinvey setup. You can find detailed instructions how to achieve that [here](https://github.com/NativeScript/nativescript-app-templates/blob/master/packages/template-master-detail-kinvey/tools/kinvey/kinvey-database-setup.md).
 
 ### Styling
-This template is set up to use SASS for styling. All classes used are based on the {N} core theme – consult the [documentation](https://docs.nativescript.org/angular/ui/theme.html#theme) to understand how to customize it. Check it out to see what classes you can use on which component.
+This template is set up to use SASS for styling. All classes used are based on the {N} core theme – consult the [documentation](https://github.com/NativeScript/theme) to understand how to customize it.
+
+It has 3 global style files that are located at the root of the app folder:
 
-It has 4 global style files that are located at the root of the app folder:
-- `_app-variables.scss` - holds the global SASS variables that are imported on each component's styles.
-- `app.scss` - the global common style sheet. These style rules are applied to both Android and iOS.
-- `platform.android.scss` - the global Android style sheet. These style rules are applied to Android only.
-- `platform.ios.scss` - the global iOS style sheet. These style rules are applied to iOS only.
+- `/app/_app-common.scss` - the global common style sheet. These style rules are applied to both Android and iOS.
+- `/app/app.android.scss` - the global Android style sheet. These style rules are applied to Android only.
+- `/app/app.ios.scss` - the global iOS style sheet. These style rules are applied to iOS only.
 
 Each component has 3 style files located in its folder:
-- `_page-name.scss` - the component common style sheet. These style rules are applied to both Android and iOS.
-- `page-name.android.scss` - the component Android style sheet. These style rules are applied to Android only.
-- `page-name.ios.scss` - the component iOS style sheet. These style rules are applied to iOS only.
+
+- `../_{page-name}-common.scss` - the component common style sheet. These style rules are applied to both Android and iOS.
```

Comment:
Why ../ ?

Reply:
Because it's not at root level, but somewhere in the project folders. Why not?

Reply:
Association with going one directory up.

Reply:
The sentence above says `Each component has 3 style files located in its folder:`,
so I think could skip `../`.

Reply:
One folder up is `..` and `/` is down, so `../{name}` is the same level. It looks weird without any indication of path, given we have put paths on all other files. If you have any suggestions of a correct indication, share.